sample.env

```diff
@@ -0,0 +1,7 @@
+MONGOHQ_URL=
+FB_CLIENT_ID=
+FB_CLIENT_SECRET=
+TWITTER_CONSUMER_KEY=
+TWITTER_CONSUMER_SECRET=
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
```

Comment:
The file contains environment variables associated with this project. This is to ensure all team members sync while working on different projects